scripts/backend/__init__.py

```diff
@@ -29,15 +29,15 @@
 
 from tiqit import *
 import fields, sys, re, utils
-from StringIO import StringIO
+from io import StringIO
 
 _backends = {}
 def initialise():
     global allFields
     allFields = fields.load()
 
     # Import all the configured backends
-    for prefix, mod in Config().section('backends').items():
+    for prefix, mod in list(Config().section('backends').items()):
```

Comment:
I assume this is 2to3 that has been wrapping all these things in lists?
Seems like an odd thing for it to do within a for loop, but maybe it's not that smart.
```suggestion
    for prefix, mod in Config().section('backends').items():
```